packages/react/src/select-value/SelectValue.tsx

```diff
@@ -0,0 +1,21 @@
+import { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { useSelectContext } from "../select-context";
+
+type SelectValueProps = BoxProps<"div", { placeholder?: string }>;
+
+export const SelectValue = forwardRef<HTMLDivElement, SelectValueProps>(
+  ({ children, placeholder, ...props }, ref) => {
+    const { downshift, itemToString } = useSelectContext("SelectValue");
```

Comment:
following up on https://github.com/optimizely-axiom/optiaxiom/pull/404/files#r1803088222

our own `selectedItem` should be passed here rather than picked from downshift

```suggestion
    const { downshift, itemToString, selectedItem } = useSelectContext("SelectValue");
```